meta/main.yml

```diff
@@ -16,6 +16,15 @@ galaxy_info:
   - name: Debian
     versions:
     - buster
+    - bullseye
+  - name: Ubuntu
+    versions:
+    - focal
+    - jammy
+  - name: Rocky Linux
+    versions:
+    - "8"
+    - "9"
```

Comment:
```suggestion
```
As far as I know, Rocky is still not in the list of supported OS'es, right? It's part of `EL`, IIRC.

Reply:
Well, now - I was too fast again. It's still listed as lint.